src/schemas/21-526EZ/schema.js

```diff
@@ -225,206 +282,233 @@ let schema = {
+                  type: 'string'
+                },
+                specialIssues: {
+                  $ref: '#/definitions/specialIssues'
+                },
+                ratedDisabilityId: {
+                  type: 'string'
+                },
+                ratingDecisionId: {
+                  type: 'string'
+                },
+                diagnosticCode: {
+                  type: 'number'
+                },
+                specialIssueTypeCode: {
+                  type: 'string'
+                },
               },
-              // Presumably, this should be an array...
-              secondaryDisabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {
-                    diagnosticText: {
-                      type: 'string'
-                    },
-                    disabilityActionType: {
-                      type: 'string'
-                    }
-                  }
-                }
-              }
+              required: ['diagnosticText', 'decisionCode', 'ratedDisabilityId']
             }
           }
-        }
+        },
+        required: ['diagnosticText', 'decisionCode', 'ratedDisabilityId']
       }
     },
-    // Presumably, this should be an array...
     treatments: {
       type: 'array',
+      maxItems: 100,
       items: {
         type: 'object',
         properties: {
-          treatment: { // Really hoping this extra nesting isn't necessary
-            type: 'object',
-            properties: {
-              treatmentCenterName: {
-                type: 'string'
-              },
-              // Can we make this in to a dateRange?
-              startTreatment: {
-                $ref: '#/definitions/date'
-              },
-              endTreatment: {
-                $ref: '#/definitions/date'
-              },
-              // Should this use a dropdown like address?
-              treatmentCenterCountry: {
-                type: 'string'
-              },
-              // Should this use a dropdown like address?
-              treatmentCenterState: {
-                type: 'string'
-              },
-              treatmentCenterCity: {
-                type: 'string'
-              },
-              treatmentCenterType: {
-                // Enum?
-                type: 'string'
-              }
-            }
+          treatmentCenterName: {
+            type: 'string',
+            maxLength: 100,
+            pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
+          },
+          // Can we make this in to a dateRange?
+          startTreatmentDate: {
+            $ref: '#/definitions/datetime'
+          },
+          endTreatmentDate: {
+            $ref: '#/definitions/datetime'
+          },
+          // Should this use a dropdown like address?
+          treatmentCenterCountry: {
+            type: 'string'
+          },
+          // Should this use a dropdown like address?
```

Comment:
Good questions, can we reuse the address definition logic here?

Reply:
Unfortunately, I doubt it. I'm guessing this will be similar to the PCIU address, but...I really don't know.